### DISCUSSION TITLE
Inquiry about enriching 5.0 Proxy features

### DISCUSSION THREAD

**Discussion by johnluoyx** [apache/rocketmq#6210] [Ideas]
We are using RocketMQ 5.0, but we encountered some problems when using the proxy, we can't create topic, get consumer & producer connections etc. through the grpc interface provided by the proxy, which causes a lot of trouble for us to get RocketMQ state.  Now we can only get some of it through the old MQAdmin, but this is not in line with the 5.0 Proxy design. Does the community have any plans to improve the proxy in the future?

**Comment by ShadowySpirits** [comment]
The introduction of admin API to the proxy is indeed necessary. PTAL @drpmma @xdkxlk

**Reply by socutes** [reply]
It seems very necessary. Is it already being done?

**Reply by drpmma** [reply]
The act of creating a topic is considered an internal cluster operation and can be fulfilled by the mqadmin tool. And the role of the proxy is more focused on exposing services
 
And getting the client connection is indeed necessary for the proxy because the connection info is stored in the proxy.

**Reply by ShadowySpirits** [reply]
> It seems very necessary. Is it already being done?

I notice someone has added GET_CONSUMER_CONNECTION_LIST request code support for proxy in #6296.
